traceapp/vis.go

```diff
@@ -14,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+// ErrTimelineItemValidation is returned by timelineItem.Valid when either
+// timelineItem.Label or timelime.ItemFullLabel are empty.
 var ErrTimelineItemValidation = errors.New("timeline item validation error")
```

Comment:
Thanks for sending this @chris-ramon !

Does this variable need to be exported at all? It seems like it is only used internally. Could it just be `errTimelineItemValidation` instead?